packages/hop-node/src/watchers/classes/BaseWatcher.ts

```diff
@@ -236,6 +236,41 @@ class BaseWatcher extends EventEmitter implements IBaseWatcher {
     }
   }
 
+  get hasCustomRoutes () {
```

Comment:
See comment above about this being unnecessary, I believe.

Reply:
Resolved by https://github.com/hop-protocol/hop/commit/4d5b5a96e8c1b000bda20ca710aaeaaab5ce568e